README.adoc

```diff
@@ -1,11 +1,22 @@
-= Address Book (Level 3)
+= Organice (Organ Transplant Manager)
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
+== Motivation
+It is undeniable that Organ Transplant is one of the greatest achievements we have.
```

Comment:
This is an eye-catching description. Here are my suggested edits:

It is undeniable that organ transplantation is one of the most life-changing medical breakthroughs in history.
		

		For a transplant to be successful, we need to have an organ that matches the receiver.
		

		Since time is precious during transplant, we aim to provide an organ matching system to efficiently match organs from donors to the patients who need them.

Reply:
Thank you so much. I wanted to show how important our ORGANice is but my phrasing does not seem to show the severity of this situation and the need for our solution. Thank you.